packages/blade/src/components/VisuallyHidden/ScreenReaderStyles.ts

```diff
@@ -0,0 +1,17 @@
+import type { CSSObject } from 'styled-components';
+
+const screenReaderStyles: CSSObject = {
+  border: 0,
+  clip: 'rect(0 0 0 0)',
+  clipPath: 'inset(50%)',
+  height: 1,
```

Comment:
1 what? What units are they mapping to in web & native?

Reply:
It would be 1px in web i know. We can be explicit.